ex02.exs

```diff
@@ -1,6 +1,25 @@
-
+#VJ
 defmodule Ex02 do
 
+  def new_counter(value) do
+    { :ok, counter } = Agent.start(fn -> value end)
```

Comment:
😭😭 I didn't see the warning in the mess of 20 lines of code returned while pushing an update to github. Now when I cleared everything and tried again, I saw it.
This seemed to fix it:
def new_counter(value) do
    _pid = Agent.start(fn -> value end)
  end

Is this the correct way to do it ?